README.md

```diff
@@ -34,9 +34,9 @@ return (
 |----------------|------------|------------|--------------------------------
 |classes         | `object` | | Override or extend the styles applied to the component.
 |closeIcon       | `node` | `<CloseIcon color={grey500} />`           | Override the close icon.
-|dataSource     | `array` | `[]` | Array of strings or nodes used to populate the list.
+|dataSource     | `array` | `[]` | Array of strings or nodes used to populate the list. (See [AutoComplete](https://material-ui.com/demos/autocomplete/))
```

Comment:
Material-UI v1 does not have an auto-complete anymore. Neither does this search field in v1. :sweat: